packages/artillery/test/scripts/scenario-payload-with-envs/scenario.yml

```diff
@@ -0,0 +1,21 @@
+#https://github.com/artilleryio/artillery/issues/1961
+config:
+  environments:
+    staging:
+      payload:
+        - fields:
+            - "id"
+            - "name"
+          skipHeader: true
+          path: "./data/my-data.csv"
+          order: sequence
+  target: https://http.cat
```

Comment:
Since the test does not depend on any particular response from the server, can we change this to use one of the mock servers we already have, or our own service, e.g. http://asciiart.artillery.io:8080. I'd rather avoid a dependency on a 3rd party service in our test suite.

Reply:
If you look at the scenario, it doesn't actually make the HTTP call, the flow just does a log instead as that's enough to prove the scenario loads the payload. 

So the target could be anything really. I'll change it to ours though! 👍